content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -553,7 +553,7 @@ jobs:
   my_first_job:
     steps:
       - name: My first step
-        uses: docker://gcr.io/cloud-builders/gradle
+        uses: docker://ghcr.io/cloud-builders/gradle
```

Comment:
"gcr" is for the Google Container Registry, which is probably the right one for "cloud-builders"

Reply:
Ho ok I was not aware that Google had a container registry. And in that case it should me mentioned somewhere in that place